msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/ltw/TestCase2572249.java

```diff
@@ -0,0 +1,142 @@
+import com.microsoft.identity.client.ui.automation.annotations.RunOnAPI29Minus;
+import com.microsoft.identity.client.ui.automation.annotations.SupportedBrokers;
+import com.microsoft.identity.client.ui.automation.app.MsalTestApp;
+import com.microsoft.identity.client.ui.automation.app.OneAuthTestApp;
+import com.microsoft.identity.client.ui.automation.broker.BrokerLTW;
+import com.microsoft.identity.client.ui.automation.broker.BrokerMicrosoftAuthenticator;
+import com.microsoft.identity.client.ui.automation.interaction.FirstPartyAppPromptHandlerParameters;
+import com.microsoft.identity.client.ui.automation.interaction.PromptParameter;
+import com.microsoft.identity.client.ui.automation.interaction.microsoftsts.MicrosoftStsPromptHandlerParameters;
+import com.microsoft.identity.labapi.utilities.client.LabQuery;
+import com.microsoft.identity.labapi.utilities.constants.TempUserType;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+//  Updated LTW, Updated Auth app and uninstall LTW
+// https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2572249
+@LTWTests
+@RunOnAPI29Minus
+@SupportedBrokers(brokers = {BrokerLTW.class})
+public class TestCase2572249 extends AbstractMsalBrokerTest {
+
+    @Test
+    public void test_2572249() throws Throwable {
+        final String username = mLabAccount.getUsername();
+        final String password = mLabAccount.getPassword();
+
+        // install updated auth app
+        final BrokerMicrosoftAuthenticator brokerMicrosoftAuthenticator = new BrokerMicrosoftAuthenticator();
+        brokerMicrosoftAuthenticator.install();
+
+        // acquire token interactively in MsalTestApp
+        final MsalTestApp msalTestApp = new MsalTestApp();
+        msalTestApp.install();
+        msalTestApp.launch();
+        msalTestApp.handleFirstRun();
+
+        final MicrosoftStsPromptHandlerParameters promptHandlerParametersMsal = MicrosoftStsPromptHandlerParameters.builder()
+                .prompt(PromptParameter.SELECT_ACCOUNT)
+                .loginHint(username)
+                .sessionExpected(false)
+                .broker(mBroker)
+                .expectingBrokerAccountChooserActivity(false)
+                .expectingProvidedAccountInBroker(false)
+                .expectingLoginPageAccountPicker(false)
+                .expectingProvidedAccountInCookie(false)
+                .consentPageExpected(false)
+                .passwordPageExpected(true)
+                .speedBumpExpected(false)
+                .registerPageExpected(false)
+                .enrollPageExpected(false)
+                .staySignedInPageExpected(false)
+                .verifyYourIdentityPageExpected(false)
+                .howWouldYouLikeToSignInExpected(false)
+                .build();
+
+        String tokenMsal = msalTestApp.acquireToken(username, password, promptHandlerParametersMsal, true);
+        Assert.assertNotNull(tokenMsal);
+
+        // uninstall LTW
+        mBroker.uninstall();
```

Comment:
In this test case, the mBroker is BrokerLTW because I set SupportedBroker annotation to BrokerLTW according to one of Fadi's comments.  If I'm going to change it back to brokerLTW.install() then brokerLTW.uninstall(), it would be against the comment that I resolved from @fadidurah 
![Screenshot 2023-08-29 at 15 58 01](https://github.com/AzureAD/microsoft-authentication-library-for-android/assets/51412053/1576e7ae-aa3d-4ee5-85fb-26e2413e5573)

Reply:
![Screenshot 2023-08-29 at 16 07 16](https://github.com/AzureAD/microsoft-authentication-library-for-android/assets/51412053/03114429-bf69-4fc0-88bc-4ac2808ae791)